### PR TITLE
try just bash uplaoder

### PR DESCRIPTION
**Pull request by stevepeak**
Debugging coverage. Please do not merge. Thanks!

### CODE REVIEW

**Comment by stevepeak** [comment]
The coverage reports produced by [slather are empty](https://codecov.s3.amazonaws.com/v4/raw/2016-07-03/E71F7BF4C697735974AD3F12E05110B8/044dcb831df0623f60ceebe35b819aadb0428e7b/485d12f1-fb2c-4534-a501-6c91ab8bc286.txt) 😦

**Comment by maxkramer** [comment]
@stevepeak thanks for testing it out, it's a pain that Slather has always worked locally, yet is incredibly temperamental when run on Travis. Ultimately there isn't a need for Slather anymore given that codecov's bash script is able to find the correct profdata report. I had a nose through the codecov (upload) bash script and it looks like sending over the `-g` parameter with a list of files to exclude, should be enough to prevent the Demo files from affecting the result of the code coverage. I'm going to update the makefile with the -g command to confirm this, and will then merge. Cheers for the support!

**Comment by maxkramer** [comment]
@stevepeak I tried using the `-g` parameter which returned the following: https://travis-ci.org/MaxKramer/Timber/builds/142311913 (build of a5842ce1be741a4ae8b56195b00c80b0009cf5fd) and uploaded all of the coverage files in the xcode project, rather than just the one for `Timber.framework.coverage.txt`. I then tried supplying the `-J Timber.framework` param to see if it would solely upload the correct file, but it didn't upload any at all. Any ideas?

File: Makefile [hunk 1]
```diff
@@ -5,9 +5,7 @@ XCODE_SCHEME = Timber-Example
 test:
 	set -o pipefail && xcodebuild test -workspace $(XCODE_WORKSPACE) -scheme $(XCODE_SCHEME) -sdk iphonesimulator9.3 | xcpretty        
 codecov:
-	PROJECT_TEMP_ROOT=$(shell xcodebuild -showBuildSettings -workspace $(XCODE_WORKSPACE) -scheme $(XCODE_SCHEME) | grep -m1 PROJECT_TEMP_ROOT | cut -d= -f2 | xargs)
-	bundle exec slather coverage --verbose -b $(PROJECT_TEMP_ROOT)
-	bash <(curl -s https://codecov.io/bash) -f ./cobertura.xml
+	bash <(curl -s https://codecov.io/bash) -J '^Timber'
```
Comment (stevepeak) [inline Makefile]: @MaxKramer I believe this is what we are looking for 👍

**Comment by maxkramer** [comment]
@stevepeak Thanks for taking another look. I've run codecov and checked the logs on the site, but its still taking into account files that exist outside of Timber/Classes which includes the demo files that I'm trying to ignore. I tried -G "Timber/Classes/*" to specify that the only files that we need code coverage for are those found in that path, but it doesn't seem to work.

https://codecov.io/gh/MaxKramer/Timber/tree/d4143079ac24bc89f11c368370e4c00326969aa3

**Comment by stevepeak** [comment]
Made another small change. Let's see if that does it 👍

**Comment by maxkramer** [comment]
@stevepeak merged! Thanks for all your help!

**Pull request merged**